src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: [name],
+    wheels: [wheels],
+    version: [version],
```

Comment:
Можна записати коротшеж
```suggestion
    name,
    wheels,
    version,
```

Reply:
мдя ....точно у нас же совпадает название переменной и ключ... спасиб :) забыл...

Reply:
make changes